alembic/env.py

```diff
@@ -95,6 +130,23 @@ def run_migrations_offline() -> None:
 
     with context.begin_transaction():
         context.run_migrations()
+        with context.begin_transaction():
+            print("-- Dropping views")
+            print(f"DROP VIEW IF EXISTS {schema_name}.visit1;")
```

Comment:
It would be really nice if we could figure out the names (and even contents) of the views from Felis, but for now this is good enough.